mobile/src/main/java/org/openhab/habdroid/ui/OpenHABAboutActivity.java

```diff
@@ -43,9 +61,21 @@ public void onCreate(Bundle savedInstanceState) {
                     .commit();
         }
 
+        try {
+            mOpenHABVersion = Integer.valueOf(getIntent().getExtras().get("openHABVersion").toString());
```

Comment:
As long as it can reach the server, yes. Why the complicated setup, though?

```
mOpenHABVersion = getIntent().getIntExtra("openHABVersion", 0);
```

Reply:
Yes, that's better